server/docker-compose.yaml

```diff
@@ -57,8 +57,8 @@ services:
   nginx:
     image: nginx:latest
     ports:
-      - "${HTTP_PORT}:80"
-      - "${HTTPS_PORT}:443"
```

Comment:
What do this change do for the ports? Is it optional for if the env var doesn't exist?

Reply:
Yeah, if the variable is unset or empty it'll default to those values. Here's the [ref](https://docs.docker.com/compose/environment-variables/) for environment variables, there's some other weird stuff we can do too